service/storage_service.cc

```diff
@@ -126,6 +126,15 @@ session_manager& get_topology_session_manager() {
     return topology_session_manager;
 }
 
+namespace {
```

Comment:
nit: I think we usually use `static` instead of anonymous namespaces, but I guess it's a matter of taste.